pygeo/mphys/mphys_dvgeo.py

```diff
@@ -6,39 +6,56 @@
 except ImportError:
     # not everyone might have openvsp installed, and thats okay
     pass
+
+try:
+    from .. import DVGeometryESP
+except ImportError:
+    # not everyone might have esp installed, and thats okay
+    pass
+
 from mpi4py import MPI
 import numpy as np
 
 # class that actually calls the dvgeometry methods
 class OM_DVGEOCOMP(om.ExplicitComponent):
     def initialize(self):
 
-        self.options.declare("ffd_file", default=None)
-        self.options.declare("vsp_file", default=None)
-        self.options.declare("vsp_options", default=None)
+        self.options.declare("geo_file", default=None)
+        self.options.declare("geo_type", default=None)
+        self.options.declare("geo_options", default=None)
 
     def setup(self):
+        self.geo_type = self.options["geo_type"]
 
         # create the DVGeo object that does the computations
-        if self.options["ffd_file"] is not None:
+        if self.geo_type == "ffd":
             # we are doing an FFD-based DVGeo
-            ffd_file = self.options["ffd_file"]
-            self.DVGeo = DVGeometry(ffd_file)
-        if self.options["vsp_file"] is not None:
-            # we are doing a VSP based DVGeo
-            vsp_file = self.options["vsp_file"]
-            if self.options["vsp_options"] is None:
+            self.DVGeo = DVGeometry(self.options["geo_file"])
```

Comment:
Also, does the FFD version not have any options that would be passed through MPhys as VSP and ESP do?